### PR TITLE
Add a usage example for tf.is_tensor

### PR DESCRIPTION
**Pull request by IrinaM21**
Added a usage example for tf.is_tensor. 

Tested on local machine with python3 ./tf_doctest.py --file=../../python/framework/tensor_util.py

### CODE REVIEW

**Comment by IrinaM21** [comment]
@googlebot I signed it!

**Comment by brightest-dazes** [comment]
@googlebot I signed it!

**Comment by brightest-dazes** [comment]
https://appengine.google.com/_ah/conflogin?continue=https%3A%2F%2Fgoto.google.com%2Fa%2Fgoogle.com%3Fcontinue%3Dhttps%253A%2F%2Fgoto.google.com%2Fprinfo%2Fhttps%253A%2F%2Fgithub.com%2Ftensorflow%2Ftensorflow%2Fpull%2F36053&authuser=0
I am receiving an error message

File: tensorflow/python/framework/tensor_util.py [hunk 1]
```diff
@@ -981,7 +981,17 @@ def is_tensor(x):  # pylint: disable=invalid-name
...
+  Usage example:  
+    >>> x = tf.constant([[1,2,3],[4,5,6],[7,8,9]])
+    >>> print(x.shape)
+    (3, 3)
+    >>> print(tf.is_tensor(x))
+    True
+    >>> y = "Hello World"
+    >>> print(tf.is_tensor(y))
+    False
+
```
Comment (HotPotatoC) [inline tensorflow/python/framework/tensor_util.py]: Hi, Please indent the code at the same level as the 'Usage Example:'

Comment (IrinaM21) [reply tensorflow/python/framework/tensor_util.py]: I have updated the indentation as you requested and re-tested it locally.

Comment (mihaimaruseac) [reply tensorflow/python/framework/tensor_util.py]: No need to use `print`, this is similar to an interactive session.

Also, no need to print the shape of the tensor.

How about

```python
>>> tf.is_tensor(tf.constant(...))
True
>>> tf.is_tensor("Hello World")
False
```

This should keep the example minimal, focused on exactly what it is documenting.

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

**Review by mihaimaruseac — DISMISSED** [review]

File: tensorflow/python/framework/tensor_util.py (hunk 1 above)
Comment (IrinaM21) [reply tensorflow/python/framework/tensor_util.py]: I have edited the usage example as you requested and re-tested it locally.

File: tensorflow/python/framework/tensor_util.py [hunk 2]
```diff
@@ -981,7 +981,13 @@ def is_tensor(x):  # pylint: disable=invalid-name
 
   If `is_tensor(x)` returns `True`, it is safe to assume that `x` is a tensor or
   can be converted to a tensor using `ops.convert_to_tensor(x)`.
-
+  
+  Usage example:  
+  >>> tf.is_tensor(tf.constant([[1,2,3],[4,5,6],[7,8,9]]))
```
Comment (mihaimaruseac) [inline tensorflow/python/framework/tensor_util.py]: You need an empty line below `Usage example`, before the doctest starts

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**